apps/theming-designer/src/components/Palette.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { Card } from '@uifabric/react-cards';
+import {
+  ThemeGenerator,
+  themeRulesStandardCreator,
+  BaseSlots,
+  FabricSlots,
+  IThemeSlotRule,
+  IThemeRules
+} from 'office-ui-fabric-react/lib/ThemeGenerator';
+
+export interface IPalette {
+  themeRules: IThemeRules;
+}
+
+export class Palette extends React.Component<{}, IPalette> {
+  render() {
+    return (
+      <Card styles={{ root: { width: '800px', height: 'auto' } }}>
+        <h1>Fabric Palette</h1>
+        <table>
+          <tr>
```

Comment:
You may want to render a `thead`, `tbody`, etc. to be more semantically correct. Refer to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table